cmd/mailer.go

```diff
@@ -43,13 +42,10 @@ func runSendMail(c *cli.Context) error {
 		}
 	}
 
-	status, message := private.SendEmail(ctx, subject, body, nil)
-	if status != http.StatusOK {
-		fmt.Printf("error: %s\n", message)
-		return nil
+	respText, extra := private.SendEmail(ctx, subject, body, nil)
+	if !extra.IsSucceeded() {
+		return handleCliResponseExtra(extra)
 	}
-
-	fmt.Printf("Success: %s\n", message)
-
+	_ = fmt.Sprintf("Sent %s email(s) to all users", respText)
```

Comment:
What's the point of this `fmt.Sprintf`?

Reply:
Do you perhaps mean
```suggestion
	_ = fmt.Printf("Sent %s email(s) to all users", respText)
```
?

Reply:
```suggestion
	_, _ = fmt.Printf("Sent %s email(s) to all users\n", respText)
```

Mistake, good catch. But it should be  `_, _ = ...`